app/views/api_docs/vendor_api_docs/reference/reference.html.erb

```diff
@@ -73,10 +73,10 @@
 
 <p class="govuk-body">
   We have a production environment and a sandbox environment.
-  When version 1.3 is launched initially for testing, it will only be accessible
+  When version <%= version %> is launched initially for testing, it will only be accessible
```

Comment:
Can we reword this sentence? Something like: `When version <%= version %> is initially launched for testing, it will only be accessible within the sandbox environment. Use the version value as specified in the following URL:`

Reply:
<img width="1053" alt="image" src="https://github.com/DFE-Digital/apply-for-teacher-training/assets/47917431/5795f881-bee7-47d9-8434-d9db42baecc2">